_includes/header.html

```diff
@@ -39,7 +39,7 @@
               <li><a href="#welcome">Welcome</a></li>
               <li><a href="#mentors">Mentors</a></li>
               <li><a href="#students">Students</a></li>
-              <li><a href="#projects">Projects</a></li>
+              <li><a href="/projects.html">Projects</a></li>
```

Comment:
There will not be any different page for Projects, there will be a section for project within this page.